tests/Fuzzed.nim

```diff
@@ -1,29 +1,52 @@
 import unittest
 
+#Util lib.
+import ../src/lib/Util
+
+#Random standard lib.
+import random
+
 const TEST_FUZZING_LOW {.intdefine.}: int = 1
 const TEST_FUZZING_MID {.intdefine.}: int = 4
 const TEST_FUZZING_HIGH {.intdefine.}: int = 8
 
+
+template setupRandom =
+    var seed: int64 = int64(getTime())
+    randomize(seed)
+    checkpoint("Randomize Seed: " & $seed)
+
+template noFuzzTest*(
+    name: string,
+    body: untyped
+) =
+    test name:
+        setupRandom
+        body
```

Comment:
This isn't a comment for this PR, more the tests overall. While I like unittest more than what we had, I'm not a fan of it for a variety of reasons. I've been debating rewriting it, especially since your Nimble script handles test discovery (which is the only reason I want to move from the custom Python harness to Pytest). As a less extreme step, would it be optimal to write a custom `suite` macro to ensure no `test`s slip in?

Reply:
I'm by no means proficient in nim to be able to write a proper macro. I might give it a try in the future but not necessarily comfortable at this point in time. Additionally, I think other things might be more pressing.

Reply:
Only reason I bring it up is the Nim tests take forever due to not being threaded. Beyond that, unittest has a lot of quirks I'm not happy with. That said, yes, it's out of scope for this and something to discuss down the line. Thanks for keeping me in check.